models/labels/addresses/infrastructure/identifier/eth_stakers/labels_eth_stakers.sql

```diff
@@ -11,31 +11,32 @@ WITH identified_stakers AS (
     , 'infrastructure' AS category
     , 'hildobby' AS contributor
     , 'query' AS source
-    , timestamp('2023-01-18') AS created_at
+    , DATE '2023-01-18'  AS created_at
     , NOW() AS updated_at
     , 'eth_stakers' AS model_name
     , 'identifier' as label_type
-    FROM {{ ref('staking_ethereum_entities')}}
+    FROM { ref('staking_ethereum_entities_legacy') } -- {{ ref('staking_ethereum_entities')}}
```

Comment:
A dunesql model cannot depend on a legacy model (/spark model)

Reply:
I may get this wrong, so `*_legacy.sql` depends on ref('*_legacy'), while dune sql depends on ref('*')?